jobs/config.json

```diff
@@ -12636,6 +12636,16 @@
       "sig-openstack"
     ]
   },
+  "pull-cluster-api-all": {
+    "args": [
+      "make",
```

Comment:
As the logic here inevitably evolves it is better to contain all logic in a script, even if it as simple as "make all". Please add a script in "./scripts/" named "ci-make-all.sh" and change the args to only contain a single value "./scripts-ci-make-all.sh".

Reply:
Note: if you choose to go with the name of "make" instead of "make-all" then it would be "./scripts/ci-make.sh".